Core.ApplicationServices/Organizations/Write/OrganizationWriteService.cs

```diff
@@ -109,6 +158,12 @@ private Result<Organization, OperationError> UpdateOrganizationPhone(Organizatio
         return organization;
     }
 
+    private Result<Organization, OperationError> UpdateOrganizationName(Organization organization, Maybe<string> name)
+    {
+        organization.Name = name.HasValue ? name.Value : null;
```

Comment:
These methods belong in the domain model (also the old ones above). Look to SystemUsageWriteService for an example
![image](https://github.com/user-attachments/assets/07089f02-04aa-41b3-97d7-fcfbaa39e857)

Reply:
I like this, didn't know we had it in Kitos